frontend/src/reducers/user.js

```diff
@@ -0,0 +1,165 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+import { USERS_URL, SESSIONS_URL } from '../urls';
+
+const initialState = {
+  login: {
+    name: localStorage.name || '',
+    userId: localStorage.name || 0,
```

Comment:
`localStorage.userID`

Reply:
I have fixed this now. Thanks for input :)